huxley/www/js/components/Huxley.js

```diff
@@ -51,24 +52,33 @@ var Huxley = React.createClass({
     var user = CurrentUserStore.getCurrentUser();
     if (User.isAnonymous(user)) {
       return (
-        <Shaker>
-          {React.cloneElement(this.props.children, { user: user })}
-        </Shaker>
+        <div>
+          <Shaker>
+            {React.cloneElement(this.props.children, { user: user })}
+          </Shaker>
+          <SupportLink />
+        </div>
       );
     } else if (User.isAdvisor(user)) {
       return (
-        <AdvisorView user={user}>
-          {React.cloneElement(this.props.children, { user: user })}
-        </AdvisorView>
+        <div>
+          <AdvisorView user={user}>
+            {React.cloneElement(this.props.children, { user: user })}
+          </AdvisorView>
+          <SupportLink />
+        </div>
       );
     } else if (User.isChair(user)) {
       return (
-        <ChairView user={user}>
-          {React.cloneElement(this.props.children, { user: user })}
-        </ChairView>
+        <div>
+          <ChairView user={user}>
+            {React.cloneElement(this.props.children, { user: user })}
+          </ChairView>
+          <SupportLink />
+        </div>
       );
     }
   },
 });
 
-module.exports = Huxley;
+module.exports = Huxley;
```

Comment:
I know Trevor said to get rid of them, but typically I like to have a newline end each file